style.css

```diff
@@ -824,4 +855,109 @@ i.fa {
   .contact-align hr {
     display: none;
   }
+
+  .d-card {
+    position: fixed;
+    height: auto;
+    width: 85vw;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    box-shadow: 0 48px 48px 48px #252f8914;
+    display: flex;
+    flex-direction: column;
+    padding: 4.5%;
+    background-color: #fff;
+    border-radius: 24px;
+  }
```

Comment:
- [ ] When I opened the popup I just saw the image and I couldn't move to see the rest, similar thing in the mobile version. Please 🙏🏽 fix this issue for a better UX. 👇🏽 

| Desktop | Mobile |
|---|---|
| ![image](https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1/assets/58642949/db3e3576-dfc6-4759-aef3-76d9163f9845) | ![image](https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1/assets/58642949/9c162a51-6f98-4af1-bc78-28d8a2768b18) |

Reply:
@franclobo 

Thanks for the review. I have done a lots of changes to get this done.
I hope this will be okay for me to get approved. And finally I will be able to move to my next project. 🥲 

> My Laptop view
![Laptop View](https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1/assets/117600135/59ac3468-1b7a-4acd-96c1-f40afede8ffc)

> 1440px according to figma
![1440pxview](https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1/assets/117600135/39ee95ab-f86f-4920-88b3-2b52677c3fdd)

> 375px according to figma
![375px view](https://github.com/Zafron047/Portfolio_Cohort-20_Mod-1/assets/117600135/306f08bd-164e-4605-a546-687788120ab0)